cmd/gmailctl/cmd/init_cmd.go

```diff
@@ -123,14 +117,13 @@ func createDefault(path, contents string) (err error) {
 	return nil
 }
 
-func resetLib() (err error) {
+func updateLib() (err error) {
```

Comment:
Please avoid named return values, as they make the function less readable (variables shadowing problems). Some parts of the codebase use it, but I should just remove them:

```
func updateLib() error {
```

Reply:
thanks for the tip